Packs/DataBee/Integrations/DataBee/DataBee.py

```diff
@@ -0,0 +1,839 @@
+                    "Hashes": [
+                        {
+                            "algorithm": hash.get("algorithm"),
+                            "fingerprint_value": hash.get("fingerprint_value"),
+                            "value": hash.get("value"),
+                        }
+                        for hash in dict_safe_get(process, ["file", "hashes"], [])
+                    ],
+                    "is_system": dict_safe_get(process, ["file", "is_system"]),
+                    "md5": dict_safe_get(process, ["file", "md5"]),
+                    "mime_type": dict_safe_get(process, ["file", "mime_type"]),
+                    "modified_time": dict_safe_get(process, ["file", "modified_time"]),
+                    "name": dict_safe_get(process, ["file", "name"]),
+                    "owner": dict_safe_get(process, ["file", "owner"]),
+                    "parent_folder": dict_safe_get(process, ["file", "parent_folder"]),
+                    "path": dict_safe_get(process, ["file", "path"]),
+                    "security_descriptor": dict_safe_get(process, ["file", "security_descriptor"]),
+                    "sha1": dict_safe_get(process, ["file", "sha1"]),
+                    "sha256": dict_safe_get(process, ["file", "sha256"]),
+                    "sha512": dict_safe_get(process, ["file", "sha512"]),
+                    "signature": dict_safe_get(process, ["file", "signature"]),
+                    "size": dict_safe_get(process, ["file", "size"]),
+                    "type": dict_safe_get(process, ["file", "type"]),
+                    "xattributes": dict_safe_get(process, ["file", "xattributes"]),
+                },
+                "name": process.get("name"),
+                "namespace_pid": process.get("namespace_pid"),
+                "parent_process": process.get("parent_process"),
+                "pid": process.get("pid"),
+                "sandbox": process.get("sandbox"),
+                "user": process.get("user"),
+            }
+            if AdditionalContext.process in additional_context
+            else {}
+        ),
+        "risk_level": data.get("risk_level"),
+        "risk_score": data.get("risk_score"),
+        "severity": data.get("severity"),
+        "start_time": data.get("start_time"),
+        "state": data.get("state"),
+        "status": data.get("status"),
+        "status_detail": data.get("status_detail"),
+        "time": data.get("time"),
+        "type_name": data.get("type_name"),
+        "User": data.get("user", {}),
+    }
+    return normalize_data
+
+
+def test_module(client: Client) -> str:
+    """
+    Test module.
+
+    Args:
+        client (Client): DataBee client.
+        params (Dict): Integration parameters.
+
+    Raises:
+        ValueError: In case of wrong request.
+
+    Returns:
+        str: Output message.
+    """
+    response = client.search(table=SearchTypes.USER, query="start_time is None", limit=1)
+    try:
```

Comment:
the call for client.search should be in try catch as this is the section that can throw an exception